kubedb.com/v1alpha2/redissentinel_test.go

```diff
@@ -0,0 +1,116 @@
+/*
+Copyright AppsCode Inc. and Contributors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha2
+
+import (
+	"testing"
+
+	mt "kmodules.xyz/resource-metrics/testing"
+
+	"github.com/google/go-cmp/cmp"
+	core "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+)
+
+func TestRedisSentinel(t *testing.T) {
+	type want struct {
+		replicas       int64
+		mode           string
```

Comment:
Is this one necessary? I mean in redis sentinel spec there is no mode inside spec?